### PR TITLE
implemented auth & drinks crud module

### PR DESCRIPTION
**Pull request by DatVu23**

### CODE REVIEW

File: src/app/src/api/.env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+DB_URI = mongodb://admin:root@localhost:27017
+DB_NAME = test
+
+JWT_SECRET = slkdfjkqjoiqjoalskdaldk
+JWT_EXPIRES = 3d
```
Comment (funylexxus) [inline src/app/src/api/.env:5]: why 3 days maybe 1 day?

**Review by funylexxus — COMMENTED** [review]
looks good, only one question

**Comment by DatVu23** [comment]
> looks good, only one question

because 3 days is standard approach

**Comment by funylexxus** [comment]
> > looks good, only one question
> 
> because 3 days is standard approach

okay fine

**Review by funylexxus — APPROVED** [review]

**Pull request merged**